klippy/extras/print_stats.py

```diff
@@ -73,6 +78,11 @@ def get_status(self, eventtime):
             'state': self.state,
             'message': self.error_message
         }
-
+    def cmd_PAUSE_STATS(self,gcmd):
+        if self.last_pause_time is None:
+            curtime = self.reactor.monotonic()
+            self.last_pause_time = curtime
+    def cmd_RESUME_STATS(self,gcmd):
+        self.note_start()
```

Comment:
It would be a good idea to make sure that RESUME_STATS only calls `note_start()` under two conditions:
1) PAUSE_STATS has been previously called
2) `self.state` has not changed since the call to PAUSE_STATS

This could be done with an attribute, for example `self.manual_pause`, that is set to True when PAUSE_STATS executes, and is set to False in `note_start()`, `note_stop()` and `note_error()`.